sdk/identity/identity/review/identity.api.md

```diff
@@ -185,6 +219,26 @@ export class ManagedIdentityCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
     }
 
+// @public
+export abstract class PersistentCredential implements TokenCredential {
```

Comment:
I like the new name better but I'm still a bit unsure about adding the all these types to the public API. Let's discuss this offline.

Reply:
I'll be pushing a fix in a bit.